src/components/ResultModal.jsx

```diff
@@ -0,0 +1,37 @@
+import React, { Component, PropTypes } from 'react';
+import Modal from 'react-modal';
+
+export default class ResultModal extends Component {
+    render() {
+        const { showModal, resultObject, closeResultModal } = this.props;
+
+        return (
+            <Modal
+                className="result-modal"
+                overlayClassName="result-modal-overlay"
+                isOpen={showModal}
+                onRequestClose={closeResultModal}
+            >
+                <div className="metadata-container">
+                    <div className="surah-header">
+                        <div className="surah-name">{resultObject.translationSurahName}</div>
+                        <div className="surah-ayah-number">
+                            {`${resultObject.surahNum}:${resultObject.ayahNum}`}
```

Comment:
Since you're within html, I don't think you need the string interpolation. You can just do `{resultObject.surahNum}:{resultObject.ayahNum}`